starter-code/css/main.css

```diff
@@ -7,4 +7,52 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.container {
+  position: absolute;
+	width: 600px;
+	height: 500px;
+	top: 150px;
+  right: 100px;
+  background: url("../images/frame.png");
+  background-size: cover;
+}
+
+.container img {
+  margin: 0 auto;
+	padding-left: 15px;
+	width: 45%;
+}
+
+.insta-logo {
+  position: relative;
+  padding: 
+}
+
+.log-in {
+  float: right;
+  font-size: 18px;
+  padding: 5px 10px;
+  border-radius: 2px;
+}
+.content {
+  width:400px;
+	position:relative;
+	left:20px;
+	top: 35px; 
+}
+
+.phone {
+  position: relative;
+	top: 100px;
+	left :200px;
+}
+
+.footer {
+  margin-top: 200px;
+  margin-left: 455px;
```

Comment:
If you are using such large numbers for a margin, then chances are there is a better way to go about it. 455 and 200 are too large. You could have just used a margin: 0 auto, to center the footer on the page.